siteapp/static/vendor/bootstrap-helpers.js

```diff
@@ -179,15 +179,8 @@ function ajax_with_indicator(options) {
     if (options.complete)
       options.complete();
 
-    if (!old_error && jqxhr.status == 500 && /^text\/html/.test(jqxhr.getResponseHeader("content-type")) && /^(<!DOCTYPE[\w\W]*>)?\s*<html/.test(jqxhr.responseText)) {
-      // We might get back HTML in a 500 error. Flask does this. Show the
-      // HTML, in an iframe.
-      show_modal_error("Error", '<iframe style="width: 100%; height: 60vh;"></iframe>')
-      var ifrm = $('#global_modal').find('iframe')[0];
-      ifrm = (ifrm.contentWindow) ? ifrm.contentWindow : (ifrm.contentDocument.document) ? ifrm.contentDocument.document : ifrm.contentDocument;
-      ifrm.document.open();
-      ifrm.document.write(jqxhr.responseText);
-      ifrm.document.close();
+    if (!old_error && jqxhr.status == 500 && /^text\/html/.test(jqxhr.getResponseHeader("content-type")) && /^(<!DOCTYPE[\w\W]*>)?\s*<html/.test(jqxhr.responseText)) {      
+      show_modal_error(jqxhr.statusText, jqxhr.responseJSON.message )
```

Comment:
I left this section in to create modals and display some sort of message in case I am missing a scenario when this would be used that I am not currently seeing.  In our specific use case, this will no longer be used.